src/runtime/components/RadioGroup.vue

```diff
@@ -71,8 +71,9 @@ export interface RadioGroupProps<T extends RadioGroupItem[] = RadioGroupItem[],
   ui?: RadioGroup['slots']
 }
 
-export type RadioGroupEmits = RadioGroupRootEmits & {
-  change: [event: Event]
+export type RadioGroupEmits<T extends RadioGroupItem[] = RadioGroupItem[], VK extends GetItemKeys<T> = 'value'> = RadioGroupRootEmits & {
+  'update:modelValue': [payload: GetModelValue<T, VK, false>]
+  'change': [event: Event]
 }
```

Comment:
I believe it should be:
```suggestion
export type RadioGroupEmits<T extends RadioGroupItem[] = RadioGroupItem[], VK extends GetItemKeys<T> = 'value'> = Omit<RadioGroupRootEmits, 'update:modelValue'> & {
  'change': [event: Event]
} & GetModelValueEmits<T, VK, false>
```

@sandros94 What do you think?

Reply:
Well, actually, there isn't much difference.

In my version, we override the field using the interface mechanism in TypeScript.
In yours, we first delete the field, then add it back.

I like both options.

Reply:
Yes it comes down to the same thing, it's only for consistency between components 😊

Reply:
> Yes it comes down to the same thing, it's only for consistency between components 😊

Also agree on the consistency side. But with all the props and emits build issues we had in the past I started preferring the mentality of first cleanup then add it back.

There is not even a performance difference as far as I know